torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2366,6 +2401,17 @@ def _make_tensor_helper(shape, low=None, high=None):
                         # These tests do not take into account custom op.get_op()
                         SkipInfo('TestCommon', 'test_variant_consistency_jit'),)
                     ),
+    OpInfo('eig',
+           op=torch.eig,
+           dtypes=floating_and_complex_types(),
+           test_inplace_grad=False,
+           supports_tensor_out=False,
```

Comment:
This will hit a logical merge conflict with https://github.com/pytorch/pytorch/pull/53259, which is landing soon. We should rebase this on top of it (where this metadata should no longer be needed).